websocket/frontend/src/todolist/TodoList.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import TodoForm from "./Todoform"
+import TodoItem from "./TodoItem"
+
+const api_url = 'http://localhost:3001/api/v1/todos/'
+
+class TodoList extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      items: []
+    }
+    this.updateTodoList = this.updateTodoList.bind(this);
+  }
+
+  componentDidMount() {
+    this.getTasks();
+  }
+
+  getTasks() {
+    fetch(api_url)
+    .then(response => response.json())
+    .then(response_items => {
+      this.setState({
+        items: response_items.reverse()
+      })
+    });
+  }
+
+  updateTodoList(item) {
+    let _items = this.state.items
+    // Unshift adds to the beginning of the array
+    _items.unshift(item)
+    this.setState({
+      items: _items
+    })
+  }
```

Comment:
```suggestion
  updateTodoList(item) {
    this.setState({
      items: [item, ...this.state.items]
    })
  }
```

Why this variable name start with underscore? What are you trying to imply here?